rest/views.py

```diff
@@ -819,7 +838,17 @@ def make_facet_counts(self, queryset):
             queryset = queryset.facet(vocabulary.id)
 
         facet_counts = queryset.facet_counts()
-        ret = get_vocabularies(facet_counts)
+        vocabs = get_vocabularies(facet_counts)
+
+        # return dictionary
+        ret_dict = {}
+
+        # process vocabularies
+        for key, values in vocabs.items():
+            missing_count = queryset.filter(
```

Comment:
Is there any potential performance impact from this? (cc @ShawnMilo) It seems like it would execute a query for every vocabulary, not sure if that's correct or if it's anything significant